impl/client/src/main/javascript/web/pentaho/ccc/visual/LineStacked.js

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright 2023 Hitachi Vantara. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "pentaho/module!_",
+  "./PointAbstract",
+  "./_trends"
+], function (module, BaseView) {
+
+  "use strict";
+
+  return BaseView.extend(module.id, {
```

Comment:
For both the view and model classes it would be good to define `LineAbstract` base classes, both centralizing common implementation and providing base configuration target modules, for shared configurations.

Also, please check the configuraton files for rules applying to the Line viz (model or view) that should also apply to `LineStacked`, and should instead target `LineAbstract`.